server/routes.js

```diff
@@ -227,6 +227,11 @@ module.exports = (app) => {
   app.get('/connected-accounts/:service/verify', aN.authenticateAppByApiKey, aN.parseJwtNoExpiryCheck, connectedAccounts.get);
 
   /**
+   * Github API - fetch all repositories using the user's access_token
+   */
+  app.get('/fetch-github-repositories', aN.authenticateAppByApiKey, aN.parseJwtNoExpiryCheck, aN.checkJwtExpiry, connectedAccounts.fetchAllRepositories);
```

Comment:
Should probably change `aN.authenticateAppByApiKey, aN.parseJwtNoExpiryCheck, aN.checkJwtExpiry` to `aN.authenticateUserByJwt()`.